src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString.split(';');
+  const result = {};
+
+  splittedString.forEach((key) => {
+    const [property, value] = key.split(':').map((element) => element.trim());
```

Comment:
The split operation is done directly within the array destructuring assignment. To improve readability, consider splitting the operation into two steps: first split the declaration into parts, then destructure it into 'property' and 'value'.